src/blueapi/config.py

```diff
@@ -166,6 +173,7 @@ class ApplicationConfig(BlueapiBaseModel):
     scratch: ScratchConfig | None = None
     oidc: OIDCConfig | None = None
     auth_token_path: Path | None = None
+    cors: CORSConfig | None = None
```

Comment:
Should: I think cors config should go under `api` rather than at the top level, it's not associated with any other part of the application.

Reply:
Yeah, that makes sense.